package.json

```diff
@@ -84,6 +84,7 @@
     "crypto-js": "4.2.0",
     "deep-object-diff": "1.1.9",
     "get-value": "3.0.1",
+    "handlebars": "^4.7.8",
```

Comment:
why we need a new package here? we can do with normal manipulation as I see size is increased with this import.

Reply:
Normal manipulation will not give the right output for template input. Down the line we have to update amplitude js sdk too to support this.